### PR TITLE
803 message dump support for buffer adapter

### LINKED ISSUE #803: Implement message dump in the buffer adapter
To reproducibly debug traffic

### PR DESCRIPTION
**Pull request by bwsw**
Now, the buffer adapter can dump all messages for later analysis.

Closes #803

### CODE REVIEW

File: savant/utils/config.py [hunk 1]
```diff
@@ -1,4 +1,33 @@
 import os
+import sys
+
+from savant.utils.logging import get_logger
+import logging
+
+logger = get_logger(__name__)
+
+
+def get_env(name):
```
Comment (tomskikh) [inline savant/utils/config.py]: Rename to `req_config` to make it similar to `opt_config`.

File: savant/utils/config.py [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
+        )
+        if logger.isEnabledFor(logging.DEBUG):
+            logger.debug(f"Available environment variables:")
+            for k, v in os.environ.items():
+                logger.debug(f"\t{k}={v}")
+            raise ValueError(
+                f"Configuration Value Not Found in environment variables: {name}"
+            )
+        else:
+            sys.exit(1)
```
Comment (tomskikh) [inline savant/utils/config.py]: Why this behavior depends on log level?

**Review by tomskikh — CHANGES_REQUESTED** [review]

File: savant/utils/config.py (hunk 1 above)
Comment (bwsw) [reply savant/utils/config.py]: fixed

File: savant/utils/config.py (hunk 2 above)
Comment (bwsw) [reply savant/utils/config.py]: fixed

**Review by tomskikh — APPROVED** [review]

**Pull request merged**